### DISCUSSION TITLE
post/spring-and-ktor/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#55] [Announcements]
# post/spring-and-ktor/

Spring 공화국에서 Spring 개발자가 Ktor로 개발해본 경험담을 공유합니다.

https://tech.kakaopay.com/post/spring-and-ktor/

**Comment by eager20** [comment]
저도 Ktor, Koin, Exposed 이용하여 현재 서비스 중에 있습니다.
DI 라이브어리로 Koin도 좋습니다. ^^

스프링에 비해 빠른 기동과 처리 속도 만족하고 있습니다. ^^
굿바이 스프링~~ ㅋㅋ

**Comment by YunNote** [comment]
안녕하세요 
Ktor을 통해 개발을 해볼까 스터디를 진행할 예정인데 
혹시 도큐먼트 말고도 참고하신 도서나 이런것이 있으실까요 ?!

**Comment by eager20** [comment]
답이 늦었네요...

당시 책을 보진 않고...
도큐먼트만 보고 작업 했었습니다.

언넝 스프링 의존성 지옥에서 빠져나오시길..

의존성 찾는거 보다.. 서드파티 라이브어리 바로 붙여서..
코틀린으로 구성하는게 더 빠름니다... ㅜㅡㅜ

2024년 8월 1일 (목) 오후 1:15, YunNote ***@***.***>님이 작성:

**Reply by oy-zzdd1558** [reply]
답변주셔서 감사합니다!
빠르게 지옥에서 나가보겠습니다 ㅎ
